packages/teleport-project-generator-gridsome/src/gridsome-project-mapping.json

```diff
@@ -0,0 +1,10 @@
+{
+  "elements": {
+    "navlink": {
+      "elementType": "router-link",
```

Comment:
is this the same mapping as vue? if so it can be skipped

Reply:
Yeah it's the same mapping 👍